neard/src/cli.rs

```diff
@@ -98,6 +103,9 @@ pub(super) enum NeardSubCommand {
     /// config)
     #[clap(name = "unsafe_reset_data")]
     UnsafeResetData,
+    /// View DB state by invoking `state-viewer`.
```

Comment:
```suggestion
    /// View DB state.
```

After this PR, `state-viewer` ceases to be a concept user needs to be aware of, it's just an impl detail.